docs/Advanced-Usage/Workloads/GAP-Benchmark-Suite.rst

```diff
@@ -35,11 +35,7 @@ To Run:
 
 .. code-block:: bash
 
-    firesim launchrunfarm    -c workloads/gapbs.ini
-    firesim infrasetup       -c workloads/gapbs.ini
-    firesim runworkload      -c workloads/gapbs.ini
-    firesim terminaterunfarm -c workloads/gapbs.ini
-
+    workloads/run-workload.sh workloads/gapbs.ini --withlaunch
```

Comment:
user already in the `workloads` dir
should just be `run-workload.sh workloads/gapbs.ini --withlaunch`